src/software/js-samples/importUsers.html

```diff
@@ -292,28 +755,54 @@ <h2>Help</h2>
 										securityGroups: securityGroups,
 										userAuthenticationType: "BasicAuthentication",
 										activeFrom: new Date().toISOString(),
-										activeTo: "2050-01-01T00:00:00.000Z"
+										activeTo: "2050-01-01T00:00:00.000Z",
+										employeeNo: employeeNo,
+										fuelEconomyUnit: fuelEconomyUnit,
+										isMetric: isMetric,
+										isLabsEnabled: isLabsEnabled,
+                                        timeZoneId: timeZoneId,
+                                        authorityName: authorityName,
+                                        authorityAddress: authorityAddress,
+                                        carrierNumber: carrierNumber,
+                                        companyName: companyName,
+                                        companyAddress: companyAddress,
+                                        licenseNumber: licenseNumber,
+                                        licenseProvince: licenseProvince,
+										isYardMoveEnabled: isYardMoveEnabled,
+										isPersonalConveyanceEnabled: isPersonalConveyanceEnabled
 									};
+
 							if (keys.length > 0){
 								newUser.driverGroups =  groups;
 								newUser.viewDriversOwnDataOnly =  false;
 								newUser.keys = keys;
 								newUser.isDriver = true;
-							}
+                            }
+                            if (licenseNumber.length > 0){
+                                newUser.driverGroups =  groups;
+								newUser.viewDriversOwnDataOnly =  false;
+                                newUser.isDriver = true;
+                            }
 							if ($(passwordResetCheckbox).is(":checked"))
 							{
 								newUser.changePassword = true;
-							}
+                            }
+                            
+							hosRulesetArray.push(hosRuleSet);
 							
 							calls.push([
 								"Add", {
 									typeName: "User",
 									entity: newUser
 								}
-							]);
+                            ]);
+                            
                         }
 						
 						api.multiCall(calls, function(results) {
+                            var userIds = results;
+							anotherCall(userIds);
```

Comment:
@ptu91 naming `anotherCall` is kind of ambiguous. Seeing as it's example code, I think it should be more descriptive.